musicHelper.md

```diff
@@ -0,0 +1,128 @@
+#Piano Music Helper
+
+###_Introduction_
+This bot helps you find a piano note of the famous word composers 
```

Comment:
notes (plural) ... written by famous composers

Reply:
Thanks for reading my terrible mistakes of english